infrastructure/main.tf

```diff
@@ -132,6 +132,18 @@ module "db" {
   common_tags  = "${var.common_tags}"
 }
 
+module "adf" {
+  source = "git@github.com:hmcts/cnp-module-adf?ref=master"
+  data_factory_name_resource_group_name = "${local.sharedResourceGroup}"
+  data_factory_name = "${local.storageAccountNameDM}"
+  input_storage_account_resource_group_name = "${local.sharedResourceGroup}"
+  input_storage_account_name = "${local.storageAccountNameDM}"
+  output_storage_account_resource_group_name = "${local.sharedResourceGroup}"
+  output_storage_account_name = "${local.storageAccountNameDM}"
```

Comment:
Are we happy keeping backups co-located in the same Storage Account and location?

Reply:
I'd say yes, there is a lock in production to the dmstoredocprod storage account so we can be assured that this won't be accidentally deleted

Reply:
if it's fine with you, it's fine with me :)